packages/big-design/src/components/AccordionGroup/hooks/useAccordion.tsx

```diff
@@ -0,0 +1,26 @@
+import { useEffect, useState } from 'react';
+
+interface UseAccordionProps {
+  defaultExpanded?: boolean;
+}
+
+export const useAccordion = ({ defaultExpanded }: UseAccordionProps) => {
```

Comment:
No need to pass in as an object, instead do:

```suggestion
export const useAccordion = (defaultExpanded: boolean) => {
```

I removed the optional because you already set `defaultExpanded` to `false` in the AccordionPanel component.